app/static/graph.js

```diff
@@ -0,0 +1,47 @@
+function busiestDayOfTheWeekGraph(events) {
+    events = JSON.parse(events);
+
+    const ctx = document.getElementById("myChart");
+    ctx.style.backgroundColor = "rgba(255, 255, 255, 1)";
+    const myChart = new Chart(ctx, {
+        type: "bar",
+        data: {
+            labels: Object.keys(events),
+            datasets: [{
+                label: "# Events",
+                data: Object.values(events),
+                backgroundColor: [
```

Comment:
I'd suggest on exporting this one to a `const` outside the function

Reply:
I changed it don't know why it isn't "outdated",
![image](https://user-images.githubusercontent.com/71080776/108188030-c51b0180-7117-11eb-832d-c14b12094e67.png)